packages/@aws-cdk/app-delivery/lib/pipeline-deploy-stack-action.ts

```diff
@@ -166,13 +166,13 @@ export class PipelineDeployStackAction extends cdk.Construct {
   }
 }
 
-function cfnCapabilities(adminPermissions: boolean, capabilities?: cfn.CloudFormationCapabilities): cfn.CloudFormationCapabilities {
+function cfnCapabilities(adminPermissions: boolean, capabilities?: cfn.CloudFormationCapabilities[]): cfn.CloudFormationCapabilities[] {
   if (adminPermissions && capabilities === undefined) {
     // admin true default capability to NamedIAM
-    return cfn.CloudFormationCapabilities.NamedIAM;
+    return [cfn.CloudFormationCapabilities.NamedIAM];
```

Comment:
I wonder if this should not contain `AutoExpand`, too.

Reply:
Yes, makes sense, I've added it. Please let me know if the changes are acceptable.